nyxx_lavalink/lib/src/node/Node.dart

```diff
@@ -0,0 +1,249 @@
+part of nyxx_lavalink;
+
+/// Represents an active and running lavalink node
+class Node {
+  /// Node options, such as host, port, etc..
+  NodeOptions options;
+
+  /// A map with guild ids as keys and players as values
+  Map<Snowflake, GuildPlayer> players = {};
```

Comment:
in that state this map would be editable, should be private with unmodifible getter